src/SDKs/SqlManagement/Management.Sql/Generated/Models/VirtualNetworkRule.cs

```diff
@@ -33,15 +33,22 @@ public VirtualNetworkRule()
         /// <summary>
         /// Initializes a new instance of the VirtualNetworkRule class.
         /// </summary>
+        /// <param name="virtualNetworkSubnetId">The ARM resource id of the
+        /// virtual network subnet.</param>
         /// <param name="id">Resource ID.</param>
         /// <param name="name">Resource name.</param>
         /// <param name="type">Resource type.</param>
-        /// <param name="virtualNetworkSubnetId">The resource ID of the virtual
-        /// network subnet</param>
-        public VirtualNetworkRule(string id = default(string), string name = default(string), string type = default(string), string virtualNetworkSubnetId = default(string))
+        /// <param name="ignoreVnetPrivateAccessConfiguration">Create firewall
+        /// rule before the virtual network has private access enabled.</param>
+        /// <param name="state">Virtual network rule state. Possible values
+        /// include: 'Initializing', 'InProgress', 'Ready', 'Deleting',
+        /// 'Unknown'</param>
+        public VirtualNetworkRule(string virtualNetworkSubnetId, string id = default(string), string name = default(string), string type = default(string), bool? ignoreVnetPrivateAccessConfiguration = default(bool?), string state = default(string))
```

Comment:
The code is not deployed in production yet. I will update the tests once that happens. The client code has been internally tested successfully.